cfme/utils/ssh_expect.py

```diff
@@ -18,3 +22,12 @@ def __init__(self, appliance):
     def answer(self, question, answer, timeout=None):
         self.expect(question, timeout=timeout)
         self.send(answer)
+
+    def expect(self, re_string, *args, **kwargs):
+        try:
+            SSHClientInteraction.expect(self, re_string, *args, **kwargs)
+        except socket.timeout:
+            current_output = '""' if not self.current_output else self.current_output
+            raise SSHExpectTimeoutError(
+                f"Timeouted when waiting for '{re_string}'. Current output:\n"
+                + textwrap.indent(current_output, '> '))
```

Comment:
optional: i would try to cover in fstring only.

Reply:
I don't think fstring does the same:
In [4]: print(textwrap.indent('aaa\nbbb', '> '))                                                          
> aaa
> bbb

Reply:
I mean rather than adding two strings

Reply:
umn, well I am not sure about putting method calls into strings. I can imagine it can fool some not too-smart static-analysis tools etc.

Reply:
@jaryn you can indeed put function calls into f-strings, but there's a fine line there of it being too messy and hard to read when there are python statements embedded into strings.  I'd allow this here, though the textwrap could be called above the raise statement, assigning it to a name and using it directly in the f-string.